src/SFA.DAS.AANHub.Application/Admins/Queries/GetAdminMemberResult.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.Admins.Queries
+{
+    [ExcludeFromCodeCoverage]
+    public class GetAdminMemberResult
+    {
+        public Guid MemberId { get; set; }
+        public string? Email { get; set; }
+        public string? Name { get; set; }
+
+        public static implicit operator GetAdminMemberResult?(Admin? admin)
```

Comment:
I do not like unnecessary null checks, is there a case where operator will be applied on null admin instance? otherwise please remove all the null checks and nullable fields.

Reply:
I have removed the nullable flags on the properties, and within the operator. I have left the checks to see if the admin object is nullable to handle 404 situations